### DISCUSSION TITLE
Welcome to Red-DiscordBot Discussions!

### DISCUSSION THREAD

**Discussion by Kowlin** [Cog-Creators/Red-DiscordBot#4411] [General]
## Welcome to the Red-DiscordBot GitHub Discussions!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Drapersniper** [comment]
My name is Draper, what I do for the community, I ensure I crash at least 2 bots per day as it keeps the doctor away

**Reply by fixator10** [reply]
> I ensure I crash at least 2 bots per day

Can confirm.

**Reply by madebylydia** [reply]
> I ensure I crash at least 2 bots per day

Can reproduce, please fix.

**Comment by TrustyJAID** [comment]
My name is TrustyJAID, I write cogs for the community and help ensure Cog Creators have a voice in how the bot is maintained while also reviewing Cog Creator applications.

**Comment by fixator10** [comment]
My name is Fixator10. I sometimes finding random bugs, sometimes PRing small fixes, and struggling to rewrite one certain cog.

**Comment by flaree** [comment]
My name is flare, I try write cogs somehow and occasionally contribute to fix bugs ~~and 1 line doc fixes~~

**Comment by Zephyrkul** [comment]
am floof

**Comment by laggron42** [comment]
My name is Laggron, I write mostly working™ cogs for server administration, and I'm also the one blamed everytime Discord has issues.

**Comment by madebylydia** [comment]
I'm the Captain, yaar! I create cogs for Red most of my time and take care of my favorite Discord Bot, Miku.
ye, just call me a weeb, I will say no, I'm not, but go ahead.

**Comment by OrcaWhale2008** [comment]
Which file has the client.run command to run the bot?

**Comment by NeuroAssassin** [comment]
Hello, I'm Neuro, and I'm the main tester for testing mention spam filters.  Oh, I also am maintainer for Red Dashboard, but that's less important.

**Comment by Jackenmen** [comment]
Hello, Jack here, Red's core dev and also the most popular virgin in Red's community 🍆

**Reply by OrcaWhale2008** [reply]
Where should I put in the bot's token to make it run? I mean a python file like main.py.

**Reply by Jackenmen** [reply]
@NotOrca22 If you need help using Red, you should ask on our support server at [discord.gg/red](https://discord.gg/red). And please stop spamming this discussion since your question is not related to it at all.

**Reply by OrcaWhale2008** [reply]
ok sry

**Comment by phenom4n4n** [comment]
I'm phen I ~~buy cogs and re-publish them~~ make shitty cogs and hot fix them mid-production.

**Comment by npc203** [comment]
Hi I'm erm- somebody who annoys the staff with audio bugs while they patiently fix and deal with me.   
I'm (at the time of writing) the newest kid in the block(hopefully) who started making cogs, and enjoying it.

Oh and i'm in the UwU OwO gang.

**Comment by owocado** [comment]


**Reply by Drapersniper** [reply]
Personally when I'm merging, I add a thank you when I remember, I have a notoriously bad memory, so it's it's more often when I interact with the PR creations in DMs and the server and reviews i usually remember.

I don't see it as a privilege at all, at least personally, I always assumed everyone contributing knew they were helping the project and the project wouldn't be what it is if the community didn't contribute.

Sorry that you feel that way, but for me at least it has nothing to do with a PR being worthy or not, if a PR is not worthy of the project it will just be rejected and not merged.

Also don't feel worried about speaking your mind, that's why discussions and the server was made in the first place, everyone's opinion does impact the project in one way or another and no one should feel like their voice isn't being heard or wouldn't matter.

**Reply by aikaterna** [reply]
Don't read into it too much. It's not based on anything more than style of the person replying.

**Reply by Twentysix26** [reply]
If your contributions weren't appreciated they would not have been merged to begin with. Every PR is, even the ones that get closed for whatever reason. We appreciate every person who takes the time to contribute to this project as they are the reason this project keeps thriving.
As this discussion is meant for introductions please consider [opening a separate discussion](https://github.com/Cog-Creators/Red-DiscordBot/discussions/new) next time.
